docs/source/troubleshooting/logs.rst

```diff
@@ -95,19 +95,15 @@ The following list outlines some of the common logs and what methods are recomme
 Note that when writing to these logs that it’s important to be aware of how they behave.
 For example, by default the ``errlog.txt`` file will persist across reboots but be rotated by ``logrotate`` very quickly due to file size while the ``messages`` log will not persist across reboots but will be rotated less often.
 
-Enabling Persistent Logs
```

Comment:
What about renaming this section to "Persistent logging", which explains what it is and has subsections talking about how to toggle it in the different versions?

Reply:
Done